assets/media-gallery.js

```diff
@@ -46,12 +46,17 @@ if (!customElements.get('media-gallery')) {
 
         this.preventStickyHeader();
         window.setTimeout(() => {
-          if (this.elements.thumbnails) {
-            activeMedia.parentElement.scrollTo({ left: activeMedia.offsetLeft });
-          }
-          if (!this.elements.thumbnails || this.dataset.desktopLayout === 'stacked') {
-            activeMedia.scrollIntoView({ behavior: 'smooth' });
+          if (this.dataset.enableStickyInfo === 'false'
+            || !this.mql.matches
+            || (this.dataset.enableStickyInfo === 'true' && this.dataset.desktopLayout === 'stacked')
+          ) {
+            const section = document.querySelector('[data-section]');
```

Comment:
Here we need a more specific selector. Otherwise it will just grab the first element in the document with that data attribute. 

So if you were to have a feat. collection with quick buy enabled underneath, then open the modal with the options, click the different variants, you get some scrolling happening in the background: [video](https://screenshot.click/26-06-5wwoq-r4u7q.mp4)

Reply:
Similarly on the home page with two feat. collection and quick add enabled on them. [video](https://screenshot.click/26-30-3z452-i5er8.mp4)